frontend/src/components/RisModForm.vue

```diff
@@ -161,11 +165,27 @@ function modTypeLabel(modType: ModType | "") {
         variant="tertiary"
         @click.prevent="$emit('generate-preview')"
       />
-      <RisTextButton
-        label="Speichern"
-        :icon="CheckIcon"
-        @click.prevent="$emit('update-mod')"
-      />
+
+      <div class="relative">
+        <RisTooltip
+          v-slot="{ ariaDescribedby }"
+          :visible="isUpdatingFinished"
+          :title="
+            updateError ? 'Fehler beim Speichern' : 'Speichern erfolgreich'
+          "
+          alignment="right"
+          attachment="top"
+          :variant="updateError ? 'error' : 'success'"
+        >
+          <RisTextButton
+            :aria-describedby="ariaDescribedby"
```

Comment:
Nitpick: in case you didn't know, Vue allows you to use a [shorthand](https://vuejs.org/guide/essentials/template-syntax.html#same-name-shorthand) for this now. Not a required change though if you prefer the more explicit version

Reply:
i didn't know (or forgot) about it thanks for mentioning it!